packages/atomic/src/components/atomic-search-box/atomic-search-box.pcss

```diff
@@ -1,2 +1,18 @@
 @import "tailwindcss/base";
 @import "tailwindcss/utilities";
+
+:host {
+  display: block;
+}
+
+.focus\:rounded-b-0:focus-within {
+  @apply rounded-t rounded-b-none
+}
+
+.suggestion[aria-selected="true"] {
+  @apply bg-light-grey
```

Comment:
Adding a comment here since the contrast ratio of the focused suggestion against other suggestion *technically* doesn't pass [success criterion 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#user-interface-components).

However, for JSUI, we ignored it for this particular case since having a correct contrast ratio would require styling that would make it way too ugly. An alternative may be to add an underline to the focused suggestion, but even that I'm not sure if it's a good idea.